src/pyats/contrib/creators/creator.py

```diff
@@ -136,6 +137,11 @@ def _parse_cli(self):
                 # Convert key to variable name
                 key = key.replace('--', '')
                 key = key.replace('-', '_')
+
+                # For argument 'verify', evaluate value of type String to type Boolean
+                if key == "verify":
+                    value = ast.literal_eval(value)
+
```

Comment:
As per your advice, I have moved my code to `Netbox.py` file inside `_generate` method. 

But instead of this:

    if self._verify != True:
        self._verify = False

I am using below code, because if `--verify=True` is passed to the argument, then as per your suggested code, it will result to `False`. 

    if isinstance(self._verify, str):
        self._verify = True if self._verify == "True" else False

Kindly let me know, if you have any suggestions.